packages/expo/src/ScreenOrientation/ScreenOrientation.ts

```diff
@@ -170,53 +146,40 @@ export async function doesSupportAsync(orientationLock: OrientationLock): Promis
   return await supportsOrientationLockAsync(orientationLock);
 }
 
+/* Make this a getter for jest */
```

Comment:
/* -> // unless you want a docblock

Also the comment isn't so clear unless you're thinking a few steps ahead, something like: `Determine the event name lazily so Jest can set up mocks in advance`